contracts/sol6/fprConversionRates/QtyStepConversionRates.sol

```diff
@@ -0,0 +1,429 @@
+pragma solidity 0.6.6;
```

Comment:
I don't see any events here, is it expected?

Reply:
I forgot because the sol4 version of fpr conversion rate uses no event.

Using events consumes more gas when setting rate and all setting rate interaction is from operator, so they can track the rate by themself.

I will add events for listing new token and config operations